addons/website/views/snippets/s_image_gallery.xml

```diff
@@ -2,18 +2,18 @@
 <odoo>
 
 <template id="s_image_gallery" name="Image Gallery">
-    <section class="s_image_gallery o_slideshow pt24 pb24 s_image_gallery_controllers_outside s_image_gallery_controllers_outside_arrows_right s_image_gallery_indicators_dots s_image_gallery_arrows_default o_image_popup" data-vcss="002" data-columns="3">
-        <div class="o_container_small overflow-hidden">
+    <section class="s_image_gallery o_slideshow pt24 pb24 s_image_gallery_controllers_outside s_image_gallery_controllers_outside_arrows_right s_image_gallery_indicators_dots s_image_gallery_arrows_default o_image_popup o_disable_snippet_insertion" data-vcss="002" data-columns="3">
+        <div class="o_container_small overflow-hidden o_disable_typing">
```

Comment:
I think we can use existing class `o-contenteditable-false` instead of introducing a new one. 
Don't forget to change it everywhere.